src/script/finale.ts

```diff
@@ -15,6 +22,22 @@ document.addEventListener("DOMContentLoaded", () => {
 }, false);
 
 let originPath = "";
+
+function LoadDOM(query : string) {
+    const viewObj = new ViewFactory();
+    const htmlDOM = viewObj.generate(Main.viModel);
+    if (htmlDOM) {
+        const url = document.location.search + document.location.hash;
+        HistoryManager.UpdateHistory(url, Main.viModel);
+        const finale = document.getElementById("divLog");
+        while (finale.hasChildNodes()) {
+            finale.removeChild(finale.lastChild);
+        }
+    finale.appendChild(htmlDOM);
+    top.postMessage({ highlight: true, id: query.slice(3).trim() }, "*");
```

Comment:
`query.slice(3)` could _probably_ be replaced by Main.viPath, I'm not a 100% sure, you'll need to test it.

Reply:
`Main.viPath` is not used anywhere. So should I use it?

Reply:
If it has the right value, then yes.

Reply:
I'm assuming Main.viPath will be null, just like Main.viObj, closing this comment.